release/src/Upload.hs

```diff
@@ -371,8 +371,9 @@ logStatusRetry shouldRetry _ status =
     else
         $logDebug ("Aborting staging repository check after " <> (tshow $ rsCumulativeDelay status) <> "µs.")
 
+-- Retry for 10 minutes total, doubling delay starting with 200ms.
 uploadRetryPolicy :: RetryPolicy
-uploadRetryPolicy = limitRetriesByCumulativeDelay (60 * 1000 * 1000) (exponentialBackoff (200 * 100))
+uploadRetryPolicy = limitRetriesByCumulativeDelay (5 * 60 * 1000 * 1000) (exponentialBackoff (200 * 100))
```

Comment:
I do not understand this in relation to the comment.

Assuming the unit here is microseconds, the start of the backoff seems to be 20ms, not 200, while the total delay seems to be 5 minutes, not 10. What am I missing?

Reply:
👍